packages/runner-ct/src/app/app.scss

```diff
@@ -50,7 +50,8 @@ main.app-ct {
     align-items: center;
     justify-content: center;
 
-    a {
+    a#menu-toggle { 
+      width: 20px;
```

Comment:
I think this change is not relevant anymore. Can you remove it ?

Reply:
The menu button appears stuck to the left side when the change is in.

Reply:
If there is no width we cannot click the button in the test, that's why it's there - it should have no visual impact